lms/static/js/views/fields.js

```diff
@@ -495,6 +495,203 @@
             }
         });
 
+        FieldViews.ImageFieldView = FieldViews.FieldView.extend({
+
+            fieldType: 'image',
+
+            templateSelector: '#field_image-tpl',
+            uploadButtonSelector: '.upload-button-input',
+
+            titleAdd: gettext("upload a photo"),
+            titleEdit: gettext("change photo"),
+            titleRemove: gettext("remove"),
+
+            iconUpload: '<i class="icon fa fa-camera" aria-hidden="true"></i><span class="sr">Upload Profile Image</span>',
+            iconRemove: '<i class="icon fa fa-remove" aria-hidden="true"></i><span class="sr">Remove Profile Image</span>',
+            iconProgress: '<i class="icon fa fa-spinner fa-pulse fa-spin" aria-hidden="true"></i></i><span class="sr">In Progress</span>',
+
+            errorMessage: gettext("We've encountered an error. Refresh your browser and then try again."),
```

Comment:
I'm guessing the user sees the same error message whether the error occurs when he's adding, removing, or changing an image, correct? If so, we can use this:

An error has occurred. Refresh the page, and then try again.